cupyx/jit/_types.py

```diff
@@ -43,4 +43,4 @@ def __str__(self):
         if dtype == numpy.float16:
             # For the performance
             dtype = numpy.dtype('float32')
-        return _typenames[dtype]
+        return dtype_to_ctype[dtype]
```

Comment:
Looks like this can be replaced by `cupy.core._scalar.get_typename()`?

Reply:
It is a good idea! I will do it on this PR.